docs/user/supported_devices.rst

```diff
@@ -196,6 +196,14 @@ ipq40xx-generic
   - NBG6617
   - WRE6606  [#device-class-tiny]_
 
+ipq40xx-mikrotik
+----------------
+
+* Mikrotik
+
+  - RBSXTsqG-5acD (SXTsq 5 ac)
+  - RBDiscG-5acD (DISC Lite5 ac)
```

Comment:
```suggestion
  - DISC Lite5 ac (RBDiscG-5acD)
  - SXTsq 5 ac (RBSXTsqG-5acD)
```

Reply:
Matching the model name again.

https://github.com/openwrt/openwrt/blob/master/target/linux/ipq40xx/image/mikrotik.mk#L60

Reply:
https://mikrotik.com/product/disc_lite5_ac